src/discord/handlers/MessageHandler.js

```diff
@@ -18,12 +19,23 @@ class MessageHandler {
       return
     }
 
+    const isReply = await this.checkReply(message)
+
     this.discord.broadcastMessage({
       username: message.member.displayName,
       message: this.stripDiscordContent(message.content),
+      reply: isReply,
     })
   }
 
+  async checkReply(message) {
+    if (!message.reference) return null
+
+    const replyingTo = await message.channel.messages.fetch(message.reference.messageID)
```

Comment:
We'd need a check here to ensure that the reference message isn't deleted, and thus causing an exception when trying to fetch a message that no longer exists.